src/components/BookCard.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/books';
+
+const BookCard = ({ title, author, id }) => {
+  const dispatch = useDispatch();
+  return (
+    <div className="book">
+      <h3>{title}</h3>
+      <p>{author}</p>
+      <input
```

Comment:
- [ ] The remove functionality should also be done through a `button` as it was specified in the project requirement